src/renderer/containers/map/infoBox/index.jsx

```diff
@@ -32,8 +32,17 @@ export default class InfoBox extends React.Component {
       <div className="info-box">
         <div className="content">
           <h2>{details.name}</h2>
-          <p>{details.address1}</p>
-          <p>{details.address2}</p>
+           <a
+            href={`https://www.google.com/maps/place/${replace(
```

Comment:
can we move this href to a const at the top of the render? like this.

```
const conversion = addr => replace(/ /g, '+', addr);
const address = 'https://www.google.com/maps/place/${conversion(details.address1)}+${conversion(details.address2}+${details.zipCode}'


....
<a href={address} ...>
```

Reply:
Sure!